### ISSUE TITLE
[FEATURE] TablesNamesFinder does not support CREATE VIEW

### ISSUE THREAD

**Issue by cptnricard** [JSQLParser/JSqlParser#2123]
Is there a reason why `TablesNamesFinder` supports a `CREATE TABLE` but throws an exception for a `CREATE VIEW`?
Wouldn't the implementation be almost identical?

The current implementation is 
```java
    @Override
    public <S> Void visit(CreateView createView, S context) {
        throwUnsupported(createView);
        return null;
    }
```

It seems that it could easily be replaced with 
```java
    @Override
    public <S> Void visit(CreateView create, S context) {
        visit(create.getView(), null);
        if (create.getSelect() != null) {
            create.getSelect().accept((SelectVisitor<?>) this, context);
        }
        return null;
    }
```

Which is almost a copy&paste of the CREATE TABLE implementation
```java
    @Override
    public <S> Void visit(CreateTable create, S context) {
        visit(create.getTable(), null);
        if (create.getSelect() != null) {
            create.getSelect().accept((SelectVisitor<?>) this, context);
        }
        return null;
    }
```

**Comment by manticore-projects** [comment]
Greetings!
You are right of course and your PR was very welcome.

**Comment by manticore-projects** [comment]
fixed via #2124
